cmd/client/service.go

```diff
@@ -1,31 +1,34 @@
 package main
 
 import (
+	"flag"
 	"fmt"
-	"time"
+	"log"
 
-	sq "github.com/Masterminds/squirrel"
-	dbUtil "github.com/monstercat/golib/db"
+	"github.com/monstercat/gologx/logxhost"
 )
 
```

Comment:
this code creates:
- a private struct 
- with private variables. 

```
type service struct {
   Machine string 
   Name string 
   LastSeen time.Time
}
```

This would not allow external packages to create service, but would allow it to use the `Machine` `Name` and `LastSeen` variables. For example, the `dbUtil` package tries to ascertain the fields in a struct. If they are private, it is unascertainable by the external package `dbUtil`. 

Also, we don't use _ for variable names usually in golang (nor do we use uppercase). The standard is camelcase.

Reply:
![image](https://user-images.githubusercontent.com/48697038/101599019-e4760f80-39ad-11eb-94dc-ae63fa98421c.png)

https://golang.org/doc/effective_go.html

Reply:
But the standard here would be to have the library functions in a separate location as they can be reused. In this case, the "separate location" has already been created =)